components/BusinessPage.vue

```diff
@@ -330,9 +330,19 @@ export default {
       })
 
       this.busy = false
+
+      // Track the search.
+      this.$ga.event('search', 'submit', this.category || 'All Categories')
     },
     select(uid) {
       this.selected = uid
+
+      // Track the select.
+      const business = this.$store.getters['businesses/get'](uid)
+      const value = [business.name, business.address, business.postcode].join(
```

Comment:
In retrospect, wondering why we don't just have the business name here... but moot point, we should keep the same format like we have here.

Reply:
I wondered too...